features/aliases.feature

```diff
@@ -126,26 +126,115 @@ Feature: Create shortcuts to specific WordPress installs
     When I run `wp eval --skip-wordpress 'echo realpath( getenv( "RUN_DIR" ) );'`
     Then save STDOUT as {TEST_DIR}
 
-    When I run `wp cli alias`
+    When I run `wp cli alias list`
     Then STDOUT should be YAML containing:
       """
       @all: Run command against every registered alias.
       @foo:
         path: {TEST_DIR}/foo
       """
 
-    When I run `wp cli aliases`
+    When I run `wp cli alias list --format=json`
+    Then STDOUT should be JSON containing:
+      """
+      {"@all":"Run command against every registered alias.","@foo":{"path":"{TEST_DIR}/foo"}}
+      """
+
+  Scenario: Get alias information
+    Given a WP installation in 'foo'
+    And a wp-cli.yml file:
+      """
+      @foo:
+        ssh: user@host:/path/to/wordpress
+      """
+
+    When I run `wp cli alias get @foo`
+    Then STDOUT should contain:
```

Comment:
Tests can be strict here. See below for the removal of the leading space that breaks this.
```suggestion
    Then STDOUT should be:
```